### ISSUE TITLE
Big lists of the expected values of different EA projects

### ISSUE THREAD

**Issue by OAGr** [QURIresearch/potential-projects#19]
It would be really great to have some sort of rating for most EA projects. This would include:

- Blog Posts 
- Articles 
- Projects 
- Events 

It's not clear what the units should be. Some ideas include:
- MicroQarps (quality adjusted research papers, domain specified) 
- Microtopias / microdooms
- Unitless (comparisons)

In the beginning, these estimates could be made by a small team. Later on, they could be made by forecasters, predicting what evaluators would say.

**Comment by uvafan** [comment]
Nuno's previous experiments in this vein seem relevant to prioritizing this.

**Comment by OAGr** [comment]
Yep, he's been doing some work in this broad area for a while now. One of the hopes was that that work would be useful for something like this. (I've been thinking about it for a while)
